lib/mutant/result.rb

```diff
@@ -11,24 +11,11 @@ module Coverage
       #
       # @api private
       def coverage
-        return Rational(0) if amount_mutation_results.zero?
+        return Rational(1) if amount_mutation_results.zero?
```

Comment:
Given that there is very little code in this method I wonder if it would be cleaner to write it without a guard clause?